packages/react/src/index.d.ts

```diff
@@ -3,1160 +3,1121 @@
+        Info: React.FC<BasicProps>;
+        Context: React.FC<BasicProps>;
+        Notice: React.FC<BasicProps>;
+    };
+
+    export interface CollapseProps extends BasicProps {
+        eventKey?: string;
+        timeout?: number;
+        dimension?: 'height' | 'width' | FuncType;
+        getDimensionValue?: (
+            dimension: Pick<CollapseProps, 'dimension'>,
+            elem: React.ReactElement,
+        ) => number;
+    }
+    export const Collapse: React.FC<CollapseProps>;
+
+    export interface ComposerInputProps extends TextareaAutosizeProps {
+        className?: string;
+    }
+    export interface ComposerProps extends BasicProps {
+        inputProps?: ComposerInputProps;
+        attachButtonProps?: object;
+        sendButtonProps?: object;
+        sendButtonActive?: boolean;
+        disabledSendButton?: boolean;
+        disabledAttachButton?: boolean;
+        tooltipAttachButton?: string | FuncType;
+        tooltipSendButton?: string | FuncType;
+        sendButtonIcon?: IconType | FuncType;
+    }
+    export const Composer: React.FC<ComposerProps>;
+
+    export interface CounterProps extends BasicProps {
+        variant?:
+            | 'primary'
+            | 'secondary'
+            | 'accent'
+            | 'information'
+            | 'warning'
+            | 'positive'
+            | 'negative'
+            | 'white';
+
+        label?: string | FuncType;
+        number?: string | FuncType;
+        iconLeft?: IconType | FuncType;
+    }
+    export const Counter: React.FC<CounterProps>;
+
+    export interface DropdownButtonProps extends BasicWithAsProps {
+        caret?: Pick<IconProps, 'size'>;
+    }
+    export interface DropdownContainerProps extends BasicWithAsProps {
+        popperConfig?: PopperOptions;
+        additionalStyles?: React.CSSProperties;
+        flip?: boolean;
+        shouldUsePopper?: boolean;
+        rootCloseEvent?: string;
+    }
+    export interface DropdownToggleProps extends BasicProps {
+        disabled?: boolean;
+    }
+    export interface DropdownProps extends BasicWithAsProps {
+        drop?: 'up' | 'down' | 'left' | 'right';
+        flip?: boolean;
+        show?: boolean;
+        alignRight?: boolean;
+        onToggle?: () => void;
+    }
+    export const Dropdown: React.FC<DropdownProps> & {
+        Item: React.FC<BasicProps>;
+        Container: React.FC<DropdownContainerProps>;
+        Button: React.FC<DropdownButtonProps>;
+        Toggle: React.FC<DropdownToggleProps>;
+    };
+    export function useToggle(): [
+        {
+            ref: () => void;
+            'aria-haspopup': boolean;
+            'aria-expanded': boolean;
+        },
+        {
+            show: boolean;
+            toggle: () => void;
+        },
+    ];
+
+    export interface EmptyStateProps extends BasicProps {
+        name?: string;
+        src?: string;
+        alt?: string;
+        width?: number;
+        height?: number;
```

Comment:
These two can receive string value too

Reply:
Please see implemented of EmptyState, width and height will be paste to img tag not style Ref: https://www.w3schools.com/tags/tag_img.asp